### PR TITLE
fix: 2D axes labels disappears when max zoom in. #2229

### PR DESCRIPTION
**Pull request by nilscb**

### CODE REVIEW

File: typescript/packages/subsurface-viewer/src/storybook/layers/__image_snapshots__/subsurfaceviewer-axes2dlayer--colored-text-and-back-ground.png
Comment (hkfb) [inline typescript/packages/subsurface-viewer/src/storybook/layers/__image_snapshots__/subsurfaceviewer-axes2dlayer--colored-text-and-back-ground.png:1]: Looks like the tick marks have disappeared in some cases

**Review by w1nklr — CHANGES_REQUESTED** [review]
Like Havard noticed, ticks disappeared...

**Comment by w1nklr** [comment]
BTW, the current PR does not fix the issue :/

**Comment by w1nklr** [comment]
Nope, it does not work fine on storybook.
BTW, did you try to reproduce the issue prior to your PR ?
Your environment might not be able to reflect the bug, as for the the color scale with tabs ?

**Comment by nilscb** [comment]
This is the defect i filed and which this PR fixes:
https://github.com/equinor/webviz-subsurface-components/issues/2229

File: typescript/packages/subsurface-viewer/src/storybook/layers/__image_snapshots__/subsurfaceviewer-axes2dlayer--colored-text-and-back-ground.png
Comment (w1nklr) [reply typescript/packages/subsurface-viewer/src/storybook/layers/__image_snapshots__/subsurfaceviewer-axes2dlayer--colored-text-and-back-ground.png:1]: actually, after a fresh test, it's working fine.

**Comment by w1nklr** [comment]
> Nope, it does not work fine on storybook. BTW, did you try to reproduce the issue prior to your PR ? Your environment might not be able to reflect the bug, as for the the color scale with tabs ?

After a fresh test, it's working fine in storybook now.
Nevertheless, ticks still disspeared from the reference images :/

**Review by w1nklr — APPROVED** [review]

**Review by w1nklr — CHANGES_REQUESTED** [review]
Waiting for Nils' feedback

**Comment by nilscb** [comment]
I think its fine to merge this! 
I have another fix for the missing tick marks but I will make a later PR for that.

**Review by w1nklr — COMMENTED** [review]
Got the feedback from Nils.

File: typescript/packages/subsurface-viewer/src/storybook/layers/__image_snapshots__/subsurfaceviewer-axes2dlayer--colored-text-and-back-ground.png
Comment (w1nklr) [reply typescript/packages/subsurface-viewer/src/storybook/layers/__image_snapshots__/subsurfaceviewer-axes2dlayer--colored-text-and-back-ground.png:1]: They are back ;)

**Review by w1nklr — APPROVED** [review]

**Review by hkfb — APPROVED** [review]

**Review by w1nklr — APPROVED** [review]

**Pull request merged**